### PR TITLE
Thunder over nh

### PR DESCRIPTION
**Pull request by KhanSimeoni**

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -112,7 +113,7 @@ public RobotContainer() {
     //    climber, 
     //    () -> OperatorStick.getRawAxis(Constants.OperatorInputSettings.Gantry_Axis), 
     //    () -> OperatorStick.getRawAxis(Constants.OperatorInputSettings.Climb_Axis)));
-    intake.setDefaultCommand(new PurgeIntake(intake));
+    //intake.setDefaultCommand(new PurgeIntake(intake));  // Uncomment this if you want the intake to default purge
```
[the remark this answers is not in the document]
Comment (KenwoodFox) [reply src/main/java/frc/robot/RobotContainer.java]: Default purging may not be something we actually want in comp, can we discuss this sometime?

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 2]
```diff
@@ -81,7 +81,7 @@ public void flyByWireA(double steerage, double thro) {
    */
   public void flyByWireB(double steerage, double thro) {
     double dampenedThro = 1 * Math.pow(thro, 3);
-    double dampenedSteerage = 1 * Math.pow(steerage, 3);
+    double dampenedSteerage = 0.4 * Math.pow(steerage, 3);
```
[the remark this answers is not in the document]
Comment (jazzyjazz654321) [reply src/main/java/frc/robot/subsystems/Drivetrain.java:84]: wahhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh:D

[the remark this answers is not in the document]
Comment (KenwoodFox) [reply src/main/java/frc/robot/subsystems/Drivetrain.java:84]: > wahhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhh:D

Yeah lol, he is baby.

File: src/main/java/frc/robot/Constants.java [hunk 3]
```diff
@@ -87,9 +87,9 @@
     public static final int Climb_Axis = 1;         // Configured 2/15/2020
     public static final int Gantry_Axis = 0;        // Congigured 2/15/2020
 
-    public static final int Intake_Button = 1;               // Configured never
-    public static final int Purge_Button = 2;                // Configured never
-    public static final int Intake_Deploy = 6; // Configured never
+    public static final int Intake_Button = 7;               // Configured never
+    public static final int Purge_Button = 6;                // Configured never
+    public static final int Intake_Deploy = 5; // Configured never
```
Comment (KenwoodFox) [inline src/main/java/frc/robot/Constants.java]: Make sure to modify the `Configured never` comment when assigning these sorts of things, helps us judge on the fly how old a change is without using `git blame`

Not a big deal though

File: src/main/java/frc/robot/RobotContainer.java [hunk 4]
```diff
@@ -101,18 +101,24 @@ public RobotContainer() {
...
     //    climber, 
     //    () -> OperatorStick.getRawAxis(Constants.OperatorInputSettings.Gantry_Axis), 
     //    () -> OperatorStick.getRawAxis(Constants.OperatorInputSettings.Climb_Axis)));
-    intake.setDefaultCommand(new PurgeIntake(intake));
+    intake.setDefaultCommand(new SpinIntake(
+        intake,
+        () -> OperatorStick.getRawAxis(Constants.OperatorInputSettings.Intake_Feed_Axis)));
+    //intake.setDefaultCommand(new PurgeIntake(
+    //    intake,
+    //    () -> OperatorStick.getRawAxis(Constants.OperatorInputSettings.Purge_Axis))); //TODO: This gets in the way of the preivious intake deafult, so purge does not work. Possible fix: converge both purge & intake functions into a single function that takes in both buttons
```
Comment (KenwoodFox) [inline src/main/java/frc/robot/RobotContainer.java:120]: This line is too long <3

**Review by KenwoodFox — CHANGES_REQUESTED** [review]
Line too long, wayyyy too long XD

**Comment by KenwoodFox** [comment]
Everybody did amazing thank you!

**Pull request merged**